src/chat.rs

```diff
@@ -544,7 +563,7 @@ impl Chat {
 /// The current state of a chat.
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 #[non_exhaustive]
-pub struct ChatInfo {
+pub struct ChatlistItem {
```

Comment:
Why the need to rename this?

Reply:
Let me elaborate a little.  My vague plan wrt cleaning up the rust API was to at some point replace the `Chat` struct with this one, which would mean adding relevant methods to this struct and renaming this struct to `Chat`.

Secondly, this struct can exist independently from a chatlist, you can load it directly from the database using the chat_id.  So this name is a bit weird to me.

Reply:
This struct contains all the information that we need to render a ChatListItem, that's the reason why I name it like this. ChatInfo is quite vague in my opinion, but this api has a specific use case.